test/cases/load_test.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+from locust import HttpUser, task, tag, events
+
+FAIL_RATIO = float(os.getenv("LT_FAIL_RATIO", "0.01"))
+MEDIAN_RESPONSE_TIME = int(os.getenv("LT_MEDIAN_RESPONSE_TIME", "280"))
+PERCENTILE_95_RESPONSE_TIME = int(os.getenv("LT_PERCENTILE_95_RESPONSE_TIME",
+    "500"))
+
+class LoadTestImageBuilder(HttpUser):
+    """
+    Perform the load testing of image builder
+    """
+
+    def on_start(self):
+        self.client.proxies = {
+                "http":"http://squid.corp.redhat.com:3128/",
+                "https":"http://squid.corp.redhat.com:3128/"
+                }
+
+    @task
+    def test_packages(self):
+        """
+        Test the packages endpoint by making it search for qemu
+        """
+        self.client.get("/packages?distribution=rhel-84"
+                "&architecture=x86_64&search=qemu")
+
+    @task
+    def test_distributions(self):
```

Comment:
This 1 task sends 2 http requests, does locust measure per task or per request?

Reply:
Yep it does this and I find it magical :)